config/install/user.role.az_content_admin.yml

```diff
@@ -78,6 +79,8 @@ permissions:
   - 'delete webform submissions own node'
   - 'edit any webform'
   - 'edit any webform submission'
+  - 'translate any entity'
+  - 'translate editable entities'
```

Comment:
Question: Do we need (or want) these permissions on both the content editor and content admin roles?